code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -295,4 +295,16 @@
 	UnregisterSignal(holding, COMSIG_QDELETING)
 	holding = null
 
+/// Check whether the machine has Hypernob crystal inserted
+/obj/machinery/portable_atmospherics/proc/has_nob_crystal()
+	return nob_crystal_inserted
+
+/// Check whether the machine has reaction supression enabled
+/obj/machinery/portable_atmospherics/proc/reaction_suppression_enabled()
+	return suppress_reactions
```

Comment:
why are these procs?

Reply:
I understand why you need a setter, but why do we need getters? can't you just also set the var on the portable atmos machine?

Reply:
I'd need to set it for both the machine and internal tank then. And when toggled in UI, I'd need to update both of them too.